app/src/main/res/layout/navigation_bar_navigation.xml

```diff
@@ -154,4 +153,19 @@
             android:tooltipText="@string/hamburger_menu_tooltip"
             app:privateMode="@{viewmodel.isPrivateSession}"/>
     </LinearLayout>
+
+    <!-- Horizontal Divider -->
+        <View
```

Comment:
I suggest adding a transparent container around the horizontal bar, so it is easier to hit. Something similar to this:

```xml
        <FrameLayout
            android:clickable="true"
            android:id="@+id/moveBar"
            android:radius="4dp"
            android:layout_margin="10dp"
            android:layout_width="128dp"
            android:layout_height="32dp"
            android:tooltipText="@string/move_tooltip"
            android:layout_gravity="center_horizontal"
            app:visibleGone="@{viewmodel.isCurved}">

            <View
                android:layout_width="@dimen/move_bar_width"
                android:layout_height="8dp"
                android:layout_gravity="top|center_horizontal"
                android:background="@drawable/rounded_line"
                android:radius="4dp" />

        </FrameLayout>
```

Reply:
Done